### PR TITLE
[BD-6] OEP-18 Compliance and Python 3.8 usage

### PR DESCRIPTION
**Pull request by ericfab179**
Update Dockerfile to use ubuntu:focal, in order to be able to use python 3.8 as default python version.

Add requirements folder. Update upgrade requirements commands. Inlude new requirements files. Remove old requirements file.

**Reviewers**
- [x] @awais786 
- [x] @morenol

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ericfab179! I've created [BLENDED-392](https://openedx.atlassian.net/browse/BLENDED-392) to keep track of it in Jira.   More details are on the [BD-06](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1594000004/BD-06+Python+3.8+Upgrade) project page.


When this pull request is ready, tag your edX technical lead.

**Review by morenol — APPROVED** [review]

**Review by awais786 — APPROVED** [review]

**Comment by awais786** [comment]
@jmbowman Please review and merge if every things seems fine. I don't have merging rights on this repo.

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 # Set the locale
 RUN locale-gen en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
-WORKDIR /edx/app/hermes
-ADD requirements.txt /edx/app/hermes/requirements.txt
-RUN pip install -r /edx/app/hermes/requirements.txt
+WORKDIR /edx/app/hermes/requirements/
```
Comment (jmbowman) [inline Dockerfile]: I don't think you'd need the different workdir if you prefixed the pip install file paths below with `requirements/`.

**Review by jmbowman — COMMENTED** [review]
Looks reasonable to me, creating a ticket for SRE to review also.

**Comment by nadeemshahzad** [comment]
Please correct the requirement file path in Makefile to not break the Hermes play in configuration  
 https://github.com/edx/configuration/blob/cda45d21a1dc0b5c22c6ef46f67a6bcbd9797328/playbooks/roles/hermes/tasks/main.yml#L65
```
requirements:
	pip install -r requirements.txt
```

**Review by nadeemshahzad — APPROVED** [review]

**Comment by ericfab179** [comment]
@awais786 We think this is ready to be merged.

**Comment by morenol** [comment]
@ericfab179 could you create a PR in configuration with the new python version?

https://github.com/edx/configuration/blob/e7836fc383f22dcf1653929c92301894ff19fde7/playbooks/roles/hermes/tasks/main.yml#L45

**Comment by awais786** [comment]
@nadeemshahzad @jmbowman I don't have merging rights.

**Comment by ericfab179** [comment]
@morenol PR created https://github.com/edx/configuration/pull/5889

**Comment by awais786** [comment]
We can close this PR.

**Comment by natabene** [comment]
@awais786 I don't have closing rights. Can you close?

**Comment by openedx-webhooks** [comment]
@ericfab179 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/hermes&entry.867055334=https://github.com/edx/hermes/pull/10&entry.1484655318=https://github.com/ericfab179&entry.752974735=2020-07-06+02:27&entry.1917517419=2022-09-13+17:59&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**